frontend/js/main.js

```diff
@@ -108,41 +75,26 @@ jQuery(document).on('change', '#upload', function (event) {
  */
 const createEditDataTypeTable = () => {
   globalDataTypeList = JSON.parse(localStorage.getItem('globalDataTypeList'));
-  dataTypeListLength = Object.keys(globalDataTypeList).length;
-  globalDataTypeTable = document.createElement('table');
-  globalDataTypeTable.className = 'data-type-table';
-  globalDataTypeTable.setAttribute('id', 'globalDataType');
-  tableBody = document.createElement('tbody');
-
-  tableRow = document.createElement('tr');
-  tableHeader1 = document.createElement('th');
-  tableHeader1.innerHTML = 'Source';
-  tableHeader2 = document.createElement('th');
-  tableHeader2.innerHTML = 'Spanner';
-  tableRow.appendChild(tableHeader1);
-  tableRow.appendChild(tableHeader2);
-  tableBody.appendChild(tableRow);
+  let dataTypeListLength = Object.keys(globalDataTypeList).length;
+  let tableContent = '';
+  let globalDataTypeTable = '';
+  let globalDataTypeDiv = document.getElementById('globalDataType');
 
   for (var i = 0; i < dataTypeListLength; i++) {
-    tableRow = document.createElement('tr');
-    tableRow.setAttribute('id', 'dataTypeRow' + (i + 1));
+    tableContent += `<tr id='dataTypeRow${(i + 1)}'>`;
```

Comment:
This <TR> started but not closed

Reply:
this was closed at line 112 .. since now the function is moved to schema-conversion.js file, line number is changed to 311 of schema-conversion.js

Reply:
I have found below cod on line 311, I couldn't find TR close tag anywhere
```
tableContent += selectHTML + `</td>`;
```

Reply:
</tr> tag is added now